Source/RapyutaSimulationPlugins/Private/Core/RRPlayerController.cpp

```diff
@@ -53,8 +53,12 @@ bool ARRPlayerController::Initialize()
     check(ActorCommon);
     check(ActorCommon->SceneInstanceId == SceneInstanceId);
 
-    MainCamera = ActorCommon->MainCamera;
-    check(MainCamera);
+    SceneCamera = ActorCommon->SceneCamera;
+    check(SceneCamera);
```

Comment:
Do we really need these check/verify, e.g. is it possible to change to ensure? Crashing editor take time to restarting

Reply:
We could though I'd also want to clarify the meanings of `ensure` vs `check/verify`
* `ensure` acts as exceptions that should come from incorrect usage, mostly to inform users about how to interact with Sim
* `check/verify` acts like `assert`, purely dev-oriented, thus are more essential to dev to prevent early critical bugs, which in this case are required since SceneCamera is just necessary for the scene capturing.
It might be not straightforward to decide which to use, generally `check/verify` helps save time in debugging, since there is no point to go further with a nullptr variant. But yeah for sake of saving and restarting, we should switch to `ensure`.

Reply:
We could though I'd also want to clarify the meanings of `ensure` vs `check/verify`
* `ensure` acts as exceptions that should come from incorrect usage, mostly to inform users about how to interact with Sim
* `check/verify` acts like `assert`, purely dev-oriented, thus are more essential to dev to prevent early critical bugs, which in this case are required since SceneCamera is just necessary for the scene capturing.
It might be not straightforward to decide which to use, generally `check/verify` helps save time in debugging, since there is no point to go further with a nullptr variant. But yeah for sake of saving athe restart, we should switch to `ensure`.